pages/common/tailscale-up.md

```diff
@@ -0,0 +1,37 @@
+# tailscale up
+
+> Connects the client to the Tailscale network.
+> In version 1.8, command line arguments are stored and reused until they're overwritten or `--reset` is called.
```

Comment:
Is this only version 1.8, or is it 1.8 and above?

Reply:
above

Reply:
Perhaps we could update this to explain that then?

Reply:
```suggestion
> In version 1.8 and above, command line arguments are stored and reused until they're overwritten or `--reset` is called.
```